<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user</class>
 <widget class="QDialog" name="user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>661</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="buscarUser">
    <attribute name="title">
     <string>Buscar</string>
    </attribute>
    <widget class="QPushButton" name="buscarbtn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Correo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Apellidos</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Nombres</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Fecha de nacimiento</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchLine">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameLine">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>311</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="apellidoLine">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>311</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="correoLine">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>311</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="fechaLine">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>290</y>
       <width>311</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="publiUser">
    <attribute name="title">
     <string>Publicaciones</string>
    </attribute>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>20</width>
       <height>351</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha</string>
     </property>
    </widget>
    <widget class="QComboBox" name="fechaBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="fechaBtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>fecha</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ordenBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>inOrder</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>preOrder</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>postOrder</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Recorrido limitado</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recorridoBtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Recorrido</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Orden</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>210</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </widget>
    <widget class="QPushButton" name="creaPubliBtn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Crear nueva publicaci√≥n</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollPubli">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>381</width>
       <height>261</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>379</width>
        <height>259</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="AllPubliBtn_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Show All</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cantidadLine">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="soliUser">
    <attribute name="title">
     <string>Solicitudes</string>
    </attribute>
    <widget class="QTableWidget" name="tableUsuario">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>511</width>
       <height>131</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Nombres</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellidos</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Correo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha de Nacimiento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Solicitud</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="tableRecibidas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>311</width>
       <height>131</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Correo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Aceptar</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rechazar</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="tableEnviadas">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>210</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Correo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuarios</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Solicitudes Recibidas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>180</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Solicitudes Enviadas</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="reporUser">
    <attribute name="title">
     <string>Reportes</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>311</width>
       <height>121</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Fecha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Usuario</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>No. Comentarios</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="tableWidget_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>221</width>
       <height>121</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Fecha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cant. Publicacion</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Generar Reportes</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="perfilUser">
    <attribute name="title">
     <string>Perfil</string>
    </attribute>
    <widget class="QPushButton" name="modificarBtn">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>130</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Modificar Datos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eliminarBtn">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>190</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar Cuentas</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>0</y>
       <width>20</width>
       <height>351</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Apellidos</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Contrase√±a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Correo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Fecha de nacimiento</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Nombres</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>351</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="apellidoEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>351</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="correoEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>351</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="passEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>351</width>
       <height>24</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="fechaEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>351</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="logoutbtn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar Sesion</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
